python/WorkflowEngine.py

```diff
@@ -9,41 +9,76 @@ def main():
     PP_path = os.path.abspath('..')
     """Setting up logs."""
     LC.configure(PP_path)
-    logger = logging.getLogger("XML parser")
-    logger.info("Starting program...")
+    logger = logging.getLogger('XML parser')
+    logger.info('Starting program...')
     """Arg parse section."""
-    parser = argparse.ArgumentParser(description = '\n PathwayPackage [PP] is an integration platform for instantaneous processing and analysis of confocal/fluorescent microscopy images software. \n[...]')
+    parser = argparse.ArgumentParser(description = '\n PathwayPackage [PP] is '
+    'an integration platform for instantaneous processing and analysis of '
+    'confocal/fluorescent microscopy images software. \n[...]')
     parser.add_argument('-s',
                 metavar = '<settingspath>', 
                 type = str, 
                 required = True, 
                 nargs = 1, 
                 help = 'Path to input_settings (in XML format).')
+    parser.add_argument('-a',
+                metavar = '<argument>', 
+                type = str, 
+                default=False,
+                nargs = 2, 
+                help = 'Parameter <key> <value> that would be add '
+                'to config dictionary. I.e. -a number_of_cores 4')
     parser.add_argument('-m',
                 metavar = '<multiplesettings>', 
                 type = str, 
                 default=False,
                 nargs = 1, 
-                help = 'Option allows to process multiple configuration settings.')
+                help = 'Option allows to process multiple '
+                'configuration settings.')
+    parser.add_argument('-c',
+                metavar = '<config>', 
+                type = str, 
+                default=False,
+                nargs = 1, 
+                help = 'Path to additional file containing '
+                'configuration settings (in XML format).')
     args = parser.parse_args()
     """Setting up pipeline."""
-    logger.debug("Input settings source: %s", args.s[0])
+    logger.debug('Input settings source: %s', args.s[0])
     if os.path.isabs(args.s[0]): #relative or absolute path
         settings_path = FM.path_unify(args.s[0])
     else:
-        settings_path = FM.path_join(PP_path, "configuration_settings" , args.s[0])
-    if args.m != False: #single or multiple settings
+        settings_path = FM.path_join(PP_path, 
+                                     'configuration_settings' ,
+                                     args.s[0])
+    if args.a != False: # optional argument
```

Comment:
@Setnea why we cannot write ?: 
```
if args.a:
    additional_arg = {(args.a[0]) : (args.a[1])}
``` 

https://google.github.io/styleguide/pyguide.html?showone=True/False_evaluations#True/False_evaluations